doc/Reference/Markup/Styles.md

```diff
@@ -0,0 +1,75 @@
+---
+uid: Reference.Markup.Styles
+---
+# Styles
+
+Creating Styles with C# Markup is easy and intuitive using the `Style<T>` with extensions that let you easily set the value of a property the same as you might on a given control.
+
+```cs
+new Style<TextBlock>()
+	.Setters(s => s.FontSize(14))
+```
+
+## Basing a Style on another Style
+
+Sometimes you may want to base a style on another style. This can be done one of two ways. The first is that you can provide the name/key of the style. It is important to remember that this has a limitation of only working for globally accessible styles through the Application and is best used for default styles.
+
+```cs
+new Style<TextBlock>()
+	.BasedOn("GlobalStyleResourceName")
+```
+
+The second way is to provide a `Style` instance.
```

Comment:
```suggestion
Creating Styles with C# Markup is easy and intuitive using the `Style<T>` with extensions that let you easily set the value of a property the same as you might on a given control.

```cs
new Style<TextBlock>()
	.Setters(s => s.FontSize(14))
```

## Basing a Style on another Style

Sometimes you may want to base a style on another style. This can be done in one of two ways. The first is that you can provide the name/key of the style. It is important to remember that this has a limitation of only working for globally accessible styles through the Application and is best used for default styles.

```cs
new Style<TextBlock>()
	.BasedOn("GlobalStyleResourceName")
```

The second way is to provide a `Style` instance.
```